Updated Prometheus Remote write options (#906)
grafana/k6-docs: src/data/markdown/translated-guides/en/03 Results output/200 Real-time/00 Prometheus.md

<CodeGroup labels={[""]}>

```bash
    K6_PROMETHEUS_INSECURE_SKIP_TLS_VERIFY=false \
    K6_PROMETHEUS_USER=foo \
    K6_PROMETHEUS_PASSWORD=bar \
    ./k6 run script.js -o output-prometheus-remote
```

</CodeGroup>

## Options

The full list of available options that can be configured and passed to the extension:

| Name                                     | Value                                                                                                       |
| ---------------------------------------- | ----------------------------------------------------------------------------------------------------------- |
| `K6_PROMETHEUS_REMOTE_URL`               | Address of the Prometheus Remote Write endpoint. The default value is `http://localhost:9090/api/v1/write`. |

> | `K6_PROMETHEUS_HEADERS`                  | Additional headers that should be included in the HTTP requests. Optional. |

Reviewer: ```suggestion
| `K6_PROMETHEUS_HEADERS`                  | Additional to include in the HTTP requests. Optional. |
```

Author: ```suggestion
| `K6_PROMETHEUS_HEADERS`                  | Additional headers to include in the HTTP requests. Optional. |
```
